### DISCUSSION TITLE
Feature for Changing Rule Priority Order

### DISCUSSION THREAD

**Discussion by CabbageOfDIscord** [furybee/chrome-tab-modifier#281] [Ideas]
Love this extension! Makes my job so much easier. 

One thing that would be nice would be the ability to change whether the first-encountered or last-encountered rule is applied to a tab matching multiple rules. 

Currently, since every rule is added to the bottom of the list, if I want a rule to apply to all pages at a particular domain, but I also want more specific rules to apply to specified pages at that domain at-will, then I need to move every new single-page rule at that domain to a position above the domain-wide rule. 

In the following example, without manually adjusting rule order to place rule 4 over rule 1, the following order results in ALL tabs at example.com being marked "TICKETS", regardless of the newly-added url-specific rule. 

1. IF URL CONTAINS "example.com" SET TITLE "TICKETS" <--[original domain-wide rule]
2. randomrule2
3. randomrule3
4. IF URL CONTAINS "example.com/ticket/12345" SET TITLE "URGENT / J. DOE" <--[new url-specific rule]

--

I would love the option to set the list to apply the *last* matching rule, rather than the first, or an option during the addition process that accomplishes the same end result, maybe by placing the rule immediately before the first-encountered double-matched rule. 

I have little code experience so I'm unsure what the easiest way to do this would be. No pressure. 

Thanks, great work on this!

**Comment by sylouuu** [comment]
You can now reorder rules!
